cleverhans_tutorials/mnist_tutorial_tf.py

```diff
@@ -108,10 +113,10 @@ def evaluate():
             acc = model_eval(
                 sess, x, y, preds, X_test, Y_test, args=eval_params)
             report.clean_train_clean_eval = acc
-            assert X_test.shape[0] == test_end - test_start, X_test.shape
+#            assert X_test.shape[0] == test_end - test_start, X_test.shape
             print('Test accuracy on legitimate examples: %0.4f' % acc)
         model_train(sess, x, y, preds, X_train, Y_train, evaluate=evaluate,
-                    args=train_params, rng=rng, var_list=model.get_params())
```

Comment:
Please keep the var_list argument. It makes it less likely to cause global variable bugs when copy-pasting code.